lib/parser.js

```diff
@@ -15,15 +18,24 @@ Parser.prototype._handleLines = function (lines, cb) {
   for (var i = 0; i < lines.length; i++) {
     if (lines[i] === '') continue
 
-    var json
+    var err = null
```

Comment:
Setting these to `null` since `var` isn't block-scoped... Should probably ES2015-ify the entire file :)